UnifiedApi/Client/Services/Submitter/Service.cs

```diff
@@ -332,9 +347,23 @@ public string SessionId
               break;
           }
 
-          errorHandler(resultStatusData.TaskId,
-                       taskStatus,
-                       details);
+          Logger?.LogDebug("errorHandler for {taskId}, {taskStatus}: {details}",
```

Comment:
Case ? E

Reply:
I don't know, the error handler is called `errorHandler`...

Reply:
But in the client code, it is with a E, there is an interface for it

Reply:
I would put a `nameof(IIIIII.ErrorHandler)` if possible